content/en/blog/SSI-identity-provider/index.md

```diff
@@ -0,0 +1,62 @@
+
+Let's suppose that our imaginary friend Alice uses an application, say a dating service, that provides a Facebook login button for its users. Each time Alice logs in, Facebook becomes aware that Alice uses the dating service. It also learns at which time and how often Alice is using the service. Alice probably didn't want to share this data with Facebook and did this unintentionally. Even worse, Alice probably uses a similar login approach with other applications. Little by little, Facebook learns a great deal about which applications Alice is using and how often. Moreover, as applications usually provide a limited amount of login options, most users choose the biggest identity providers such as Facebook and Google. The big players end up collecting an enormous amount of data over users.
+
+## How Would SSI and Verified Data Change the Scenario?
+
+In the traditional OIDC flow, identity providers hold sensitive end-user data and personally identifiable information. Yet, this is not the case with the SSI model, where the user owns her data and stores it in her digital wallet as verifiable credentials. In the SSI-enabled authentication process, instead of typing username and password to the identity provider login form, the user presents verifiable proof of the needed data. No third parties are necessary for the login to take place.
+
+Furthermore, the transparent proof presentation process lets the user know which data fields the application sees. On the contrary, in the traditional flow, the user data usually gets transferred server-to-server invisibly. That makes the user unable to determine which data in detail the identity provider forwards from its' silo.
+
+{{< imgproc attributes Fit "625x625" >}}
+<em>In proof presentation, the wallet user sees in detail which attributes she shares with the application.</em>
+{{< /imgproc >}}
+
+The verifiable credentials technology would even allow computations on the user data without revealing it. For example, if we assume that Alice has a credential about her birthdate in her wallet, she could prove that she is over 18 without exposing her birthdate when registering to the dating service.
+
+## Midway Solution for Speeding up Adoption
+
+The ideal SSI-enabled OIDC login process wouldn't have any identity provider role. The identity provider (or any other suitable issuer) would issue the credential to the user's wallet before any logins. After the issuance, the user can use the data directly with the client applications as she wishes without the identity provider knowing it. The OIDC extension [SIOP (Self-Issued OpenID Provider)](https://openid.net/specs/openid-connect-self-issued-v2-1_0.html) tries to reach this goal. The specification defines how the client applications can verify users' credential data through the renewed OIDC protocol. Unfortunately, implementing SIOP would require considerable changes to all OIDC client applications.
```

Comment:
".. any identity provider role". I would continue: .. role, or the user is her identity provider herself.